src/components/course/data/utils.jsx

```diff
@@ -357,13 +358,17 @@ export function findHighestLevelSku({ courseEntitlements, seats }) {
   return findHighestLevelSeatSku(seats) || findHighestLevelEntitlementSku(courseEntitlements);
 }
 
+export function isActiveSubscriptionLicense(subscriptionLicense) {
+  return subscriptionLicense?.status === LICENSE_STATUS.ACTIVATED;
+}
+
 export function shouldUpgradeUserEnrollment({
   userEnrollment,
   subscriptionLicense,
   enrollmentUrl,
 }) {
   const isAuditEnrollment = userEnrollment?.mode === COURSE_MODES_MAP.AUDIT;
```

Comment:
I am wondering if its also worth extracting this out into its own function similar to `isActiveSubscriptionLicense`?

Reply:
I'm going to defer on this comment for now. Primarily because IIRC the `shouldUpgradeUserEnrollment` function is only called as part of `EnrollAction` which is a component that now deprecated and we're moving away from.